### PR TITLE
0.2 Release Review

### PR DESCRIPTION
**Pull request by bradeales**
Final review before release.

Release name: ?  Simply 0.2, or pick a codename subject e.g. CO mountains. First release name should begin with "a", alphabetically incrementing)

Release description: ? - "This is the first production release of the SCOS Transfer Specification."

### CODE REVIEW

**Comment by MichaelCotton** [comment]
Release name: Sounds like we need a category to increment alphabetically. How about classic book titles? Animal Farm?

Is it possible to include Doug's schedule entry updates in this release?

**Comment by djanderson** [comment]
I don't any reason that the new Schedule Entry API will run into snags, so I'm comfortable "spec"ing it here and then doinging the implementation to match by tomorrow.

**Comment by bradeales** [comment]
Updated ScheduleEntry object as per issue discussion. Please check the descriptions make sense / are succinct enough.

File: README.md [hunk 1]
```diff
@@ -151,9 +151,9 @@ The `ScheduleEntry` object requires the following name/value pairs:
...
-|`stop_is_relative`|false|boolean|seconds|`stop` should be interpreted as seconds after `start`. Default is false.|
-|`stop`|false|integer|seconds|Absolute stop time of the entry in [Unix time](https://en.wikipedia.org/wiki/Unix_time). If left unspecified, the scheduler MUST continue scheduling tasks until manually stopped.|
+|`start`|false|datetime|[ISO-8601](https://github.com/gnuradio/SigMF/blob/master/sigmf-spec.md#the-datetime-pair)|Requested time to schedule the first task. Default if unspecified is to start as soon as the task is received.|
+|`relative_stop`|false|integer|seconds|Relative seconds after `start` when the task will end. If both `relative_stop` and `absolute_stop` are unspecified, the task will carry on forever. Specifying both `relative_stop` and `absolute_stop` will result in an error.|
+|`absolute_stop`|false|datetime|[ISO-8601](https://github.com/gnuradio/SigMF/blob/master/sigmf-spec.md#the-datetime-pair)|Absolute time when the task will end. If both `relative_stop` and `absolute_stop` are unspecifified, the task will carry on forever. Specifying both `relative_stop` and `absolute_stop` will result in an error.|
```
Comment (djanderson) [inline README.md]: unspecifified -> unspecified

**Review by djanderson — DISMISSED** [review]

**Review by MichaelCotton — APPROVED** [review]
Looks good. Per @djanderson we should change to rfpaths from rfpath. If we go w CO mtns, audubon is the highest peak in Indian Peaks near Boulder.

**Pull request merged**